.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+on:
+  pull_request:
+    paths:
+      - .github/workflows/tests.yml
+      - ml_frameworks_bot/**
+      - requirements.txt
+      - setup.py
+
+jobs:
+  code-format:
+    name: check code format using black
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: psf/black@stable
```

Comment:
Comment: We should migrate to `ruff format` later so that we're just using a single tool.

Reply:
Good catch. You can ship this with the PyTorch retriever PR.